ViewLayer/SongPlayerView/SongPlayerView.h

```diff
@@ -1,31 +1,34 @@
 #pragma once
 #include <QObject>
+#include <QSlider>
 #include "../SongPlayerUI/I_SongPlayerUi.h"
 #include "../SongPlayerUI/SongPlayerUi.h"
+#include "../ProgressBar/ProgressBar.h"
 #include "../BusinessLayer/SongPlayer/SongPlayer.h"
 
 class SongPlayer;
 class I_SongPlayerUi;
+class ProgressBar;
 
 class SongPlayerView : public QWidget
 {
     Q_OBJECT
 
 public:
-    SongPlayerView(SongPlayer& songPlayer, I_SongPlayerUi& ui);
+    SongPlayerView(SongPlayer& songPlayer, I_SongPlayerUi& ui, ProgressBar& bar);
     ~SongPlayerView();
     void handlePlayPauseClicked();
 
 private:
     I_SongPlayerUi& ui_;
     SongPlayer& songPlayer_;
```

Comment:
The compiler warns that these are in the wrong order. They should be listed in the same order that they appear as arguments in the constructor.